### ISSUE TITLE
FS#899 - HG556a serial 31301 (model C) doesn't boot with C version fw, but boots with B version

### ISSUE THREAD

**Issue by aparcar** [aparcar/openwrt#823]
*ifsnop:*

Current firmware (17.01.2) and previous (17.01.1) don't boot in a HG56BZRB VER.A, Serial 31301xxxx. Don't boot means that router gets the default ip, and after three seconds reboots in a loop.

As per Openwrt wiki (https://wiki.openwrt.org/toh/huawei/hg556a) this serial is from the C version.

Using a 'B' firmware version makes the device boot, but without wifi.

Main difference is that B version has an atheros wifi chipset and C version has a Ralink wifi chipset. 

Also, openwrt 15.05.1 for model C boots without wifi.

So, to sum up:

Boots without wifi:
openwrt-15.05.1-brcm63xx-generic-HG556a_C-squashfs-cfe.bin
lede-17.01.2-brcm63xx-generic-HG556a-B-squashfs-cfe.bin

Stuck in a loop, (boot+reset):
lede-17.01.2-brcm63xx-generic-HG556a-C-squashfs-cfe.bin

OpenWrt Chaos Calmer 15.05.1 / LuCI 15.05-188-g87e9837 Release (git-16.043.44305-e2f9172) 

Supply the following if possible:
 - Device problem occurs on
 - Software versions of LEDE release, packages, etc.
 - Steps to reproduce
